src/main/scala/com/twitter/diffy/lifter/XmlLifter.scala

```diff
@@ -0,0 +1,33 @@
+package com.twitter.diffy.lifter
+
+import org.json4s._
+import org.json4s.native._
+;
+import scala.xml.{Elem, Node, NodeSeq}
+
+object XmlLifter {
```

Comment:
Rather than create an XmlLifter here and adding a dependency on org.json4s, please use the existing jsoup dependency and use HtmlLifter to also lift xml.